Misc/NEWS.d/next/Library/2021-12-30-21-38-51.bpo-46195.jFKGq_.rst

```diff
@@ -0,0 +1,4 @@
+Now :func:`typing.get_type_hints` does not add ``Optional`` to types with
+explicit ``None`` default value.
+
+This was changed in https://github.com/python/peps/pull/689
```

Comment:
Minor suggestion to improve clarity:
```suggestion
Now :func:`typing.get_type_hints` does not add ``Optional`` to types with
explicit ``None`` default value. This aligns to changes to PEP 484 in
https://github.com/python/peps/pull/689
```

Reply:
@Fidget-Spinner I'm totally fine with this, correct change in my opinion.

Just out of curiosity, does this change only apply to function parameters or class members too?